packages/graphql_flutter/pubspec.yaml

```diff
@@ -12,9 +12,9 @@ dependencies:
   gql_exec: ^1.0.0
   flutter:
     sdk: flutter
-  meta: ^1.7.0
```

Comment:
sorry this is painful I know, but this change need to go in a separate commit under the commit style `fix(graphql_flutter): upgrade meta,path,dart dependencies`

I am still not sure if the SDK version number is right, probably you can provide a justification for it?

Reply:
Thanks for your patience. Much appreciated. It's now a single commit (I did another rebase) with the message exactly `fix(graphql_flutter): upgrade meta,path,dart dependencies`. 

sdk requirements were also changed from 3.3 to 3.0. 

I'm not sure if you wanted two seperate PRs? Or whether it's ok now?

Reply:
can you separate the change of `graphql` inside a commit and the change of `graphql_flutter` inside another one? I use an automate tool to do the release that it is based on this kind of rules.

I know it is annoying sorry!

Reply:
Ok, I have done this. I hope it's all done correctly. Please check again @vincenzopalazzo and merge if you are happy.